db/models/Timesheet.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const db = require('../../db');
+
+const timesheetSchema = new Schema({
+  employee_id: {type: Schema.Types.ObjectId, ref: 'Employee', required: true},
+  monday: {type: Number, default: 0},
+  tuesday: {type: Number, default: 0},
+  wednesday: {type: Number, default: 0},
+  thursday: {type: Number, default: 0},
+  friday: {type: Number, default: 0},
+  saturday: {type: Number, default: 0},
+  sunday: {type: Number, default: 0},
+  weekStart: {type: String, default: null},
+  weekEnd: {type: String, default: null},
```

Comment:
I have these as date type in the database. Is that going to mess you up?  If everything is working on your side like this then I'll change it on my end.